🐛 Bug: Cannot find module '...@release-it/conventional-changelog/index.js' with pnpm

### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/release-it-action/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

This should run seamlessly in repos.

### Actual

https://github.com/JoshuaKGoldberg/create-typescript-app/actions/runs/6378928430/job/17310427310

```plaintext
Start: running release-it
ERROR Cannot find module 'file:///home/runner/work/create-typescript-app/create-typescript-app/node_modules/.pnpm/@release-it+conventional-changelog@7.0.1_release-it@16.1.5/node_modules/@release-it/conventional-changelog/index.js'
Error: Error running release-it: Error: Cannot find module 'file:///home/runner/work/create-typescript-app/create-typescript-app/node_modules/.pnpm/@release-it+conventional-changelog@7.0.1_release-it@16.1.5/node_modules/@release-it/conventional-changelog/index.js'
Error: Error: Cannot find module 'file:///home/runner/work/create-typescript-app/create-typescript-app/node_modules/.pnpm/@release-it+conventional-changelog@7.0.1_release-it@16.1.5/node_modules/@release-it/conventional-changelog/index.js'
Result from running release-it: undefined
```

### Additional Info

Linked: https://github.com/release-it/release-it/issues/1043

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,10 +1,11 @@
 import * as core from "@actions/core";
 import * as github from "@actions/github";
 import { $ } from "execa";
-import releaseIt from "release-it";
 import { shouldSemanticRelease } from "should-semantic-release";
 
 import { tryCatchInfoAction } from "./tryCatchInfoAction.js";
+
+const $$ = $({ stdio: "inherit" });
 
 export interface ReleaseItActionOptions {
 	branch: string;
@@ -36,9 +37,9 @@ export async function releaseItAction({
 		return;
 	}
 
-	await $`git config user.email ${gitUserEmail}`;
-	await $`git config user.name ${gitUserName}`;
-	await $`npm config set //registry.npmjs.org/:_authToken ${npmToken}`;
+	await $$`git config user.email ${gitUserEmail}`;
+	await $$`git config user.name ${gitUserName}`;
+	await $$`npm config set //registry.npmjs.org/:_authToken ${npmToken}`;
 
 	const octokit = github.getOctokit(githubToken);
 	const commonRequestData = {
@@ -76,7 +77,10 @@ export async function releaseItAction({
 
 	await tryCatchInfoAction("running release-it", async () => {
 		try {
-			await releaseIt({ verbose: true });
+			const { exitCode, stderr } = await $$`release-it --verbose`;
+			if (exitCode || stderr) {
+				throw new Error(stderr || `Exit code ${exitCode}.`);
+			}
 		} catch (error) {
 			core.error(`Error running release-it: ${error as string}`);
 			core.setFailed(error as string);

